CMakeLists.txt

```diff
@@ -539,7 +539,12 @@ endif()
 
 option(AVIF_BUILD_TESTS "Build avif tests." OFF)
 if(AVIF_BUILD_TESTS)
+    # Reference tests for CTest to work.
```

Comment:
Nit: What does "Reference tests" mean?

Since CTest is not as well known as CMake, it may be helpful to be a little verbose and say this allows us to use add_test() to add a test to be run by ctest, and provide the ctest command line we should use.

Reply:
Done in #846